src/alcotest-engine/test.ml

```diff
@@ -78,6 +82,15 @@ let list e =
   in
   testable (Fmt.Dump.list (pp e)) eq
 
```

Comment:
```suggestion
module Seq = struct
  include Seq
  
  (* Seq.uncons was introduced in OCaml 4.14 *)
  let uncons xs =
    match xs() with
    | Cons (x, xs) ->
        Some (x, xs)
    | Nil ->
        None
end
```

Reply:
Thanks for the review, in https://github.com/mirage/alcotest/pull/412/commits/47e5f3739a18372b818278ff415297d3dc76ee6f I rewrite `eq` without using `uncons`. Ok for you @MisterDA ?

Reply:
yep